tipsy/SettingsViewController.swift

```diff
@@ -10,41 +10,35 @@ import UIKit
 
 class SettingsViewController: UIViewController {
 
-    @IBOutlet weak var defaultTipControl: UISegmentedControl!
+    @IBOutlet private weak var defaultTipControl: UISegmentedControl!
```

Comment:
You can use accessors on an `@IBOutlet` too. Options for accessors are `open`, `public`, `internal`, `fileprivate`, and `private`. `Internal` is the default and actually never really has to be written. `Open` is used generally in libraries and frameworks. It lets you override methods and parameters. `Fileprivate` is used with extensions. We can talk about those later. Hopefully `public` and `private` are obvious.